docs-ref-conceptual/operationsmanager/index.md

```diff
@@ -4,13 +4,47 @@ description: This article describes the System Center Operations Manager REST AP
 author: mgoedtel
 ms.author: magoedte
 manager: carmonm
-ms.date: 04/26/2018
+ms.date: 04/22/2020
 ms.custom: na
 ms.prod: system-center-2016
 ms.technology: operations-manager
 ms.topic: reference
+ms.assetid: bb4bd81c-439c-479d-a045-dd5653812dff
 ---
 
 # Operations Manager REST API Reference
 
-System Center Operations Manager provides flexible and cost-effective infrastructure monitoring that helps ensure predictable performance and availability of critical applications and infrastructure services.  Operations Manager offers comprehensive monitoring for your datacenter by checking the health, performance, and availability for all monitored objects in your environment, and identify problems for rapid resolution.  For a more detailed overview, see the System Center Operations Manager product [page](https://docs.microsoft.com/system-center/scom/).  
\ No newline at end of file
+System Center Operations Manager provides flexible and cost-effective infrastructure monitoring that helps ensure predictable performance and availability of critical applications and infrastructure services. Operations Manager offers comprehensive monitoring for your datacenter by checking the health, performance, and availability of all monitored objects in your environment, and helping to identify problems for rapid resolution. For a more detailed overview, see the System Center Operations Manager [product page](https://docs.microsoft.com/system-center/scom/).
+
+> ![NOTE]
+> Operations Manager 2019 UR1 supports Cross-Site Request Forgery (CSRF) tokens to prevent CSRF attacks. If you are using Operations manager 2019 UR1, you must initialize the CSRF token. HTML scripts do not work if the CSRF tokens are not initialized.
```

Comment:
Need to fix the word "manager" in Operations manager.  Should be upper case "M".